notebooks/minigrid_colab_tutorial.ipynb

```diff
@@ -0,0 +1,558 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e-WlgAw0FDXF"
+      },
+      "source": [
+        "#  Example -  Training DQN agent on Minigrid"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "amrNQCUtqPRw"
+      },
+      "source": [
+        "In RLHive, there are config samples for DQN agents, for different environment including, Atari, Gym, Hanabi, Marlgrid, Minatar, and Minigrid. \n",
+        "\n",
+        "In this tutorial, we are using simple environments from Minigrid with different sizes like `MiniGrid-Empty-5x5-v0` and `MiniGrid-Empty-16x16-v0` which have an empty room and the objective of the agent is to reach the green square as you see in the following picture. The reward provided to the agent on reaching the green square is penalized by number of steps it took to reach there.\n",
```

Comment:
Here it would be better if you included a link to the minigrid repo. Also briefly note that the grid is partially observable.

Reply:
I have addressed this as well.